docs/connected_orgs.rst

```diff
@@ -1,14 +1,120 @@
+
+If your org has a custom domain, use the ``--login-url`` option along with the org's login url.
+
+.. code-block:: console
+
+    cci org connect <org_name> --login-url https://example.my.domain.salesforce.com
 
-Global orgs
------------
 
-Use a custom connected app
+Production and Developer Edition Orgs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No options are needed for these org types. Just run the same command you normally would to connect to a persistent org.
+
+.. code-block:: console
+
+    $ cci org connect <org_name>
+
+Sandboxes
+^^^^^^^^^
+For sandboxes, pass the ``--sandbox`` flag along with the org name.
+
+.. code-block:: console
+
+    $ cci org connect <org_name> --sandbox
+
+.. note:: The ``--sandbox`` flag can also be used for connecting a scratch org created externally to CumulusCI.
+
+Verify Your Connected Orgs
 --------------------------
+
+Run ``cci org list`` to see your org listed under the "Connected Org" table.
+
+    Example: A single persistent org connected to CumulusCI with the name ``devhub``.
+
+.. code-block:: yaml
+
+    $ cci org list
+
+    ┌Scratch Orgs──────────────┬─────────┬──────┬─────────┬──────────────┬────────┐
+    │ Name                     │ Default │ Days │ Expired │ Config       │ Domain │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ dev                      │         │ 7    │ X       | dev          │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ feature                  │         │ 1    │ X       | feature      │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ prerelease               │         │ 1    │ X       | prerelease   │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ qa                       │         │ 7    │ X       | qa           │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+    │ release                  │         │ 1    │ X       | release      │        │
+    ├──────────────────────────┼─────────┼──────┼─────────┼──────────────┼────────┤
+
+    ┌Connected Orgs────┬──────────────────────────────┬────────────┐
+    │ Name   │ Default │ Username                     │ Expires    │
+    ├────────┼─────────┼──────────────────────────────┼────────────┤
+    │ devhub │         │ peter.gibbons@initech.devhub │ Persistent │
+    └────────┴─────────┴──────────────────────────────┴────────────┘
+
+Make sure that CumulusCI can login to the connected org.
+
+.. code-block:: console
+
+    $ cci org browser <org_name>
+
+
+
+Global Orgs
+-----------
+By default, ``cci org connect`` stores the OAuth credentials for connected orgs in a *project specific* keychain. These credentials ensure that an org connected in Project A's directory isn't available when you're working in Project B's directory.
```

Comment:
"These credentials" -- did I phrase that correctly?

Reply:
@justinvholt That's a fine way to refer to the credentials (which are typically a refresh token and an access token), but I would rephrase this as "Using a project-specific keychain means that an org connected in Project A's directory isn't available when you're working in Project B's directory." In other words, keeping the two projects' orgs separate is a property of which keychain is used (i.e. where the credentials are stored). "These credentials ensure" make it sound like it's an inherent property of the credentials themselves, which is not the case.